src/app/store.ts

```diff
@@ -26,3 +26,8 @@ export const store = configureStore({
     modalActionMenusVisibility: modalActionMenusVisibilityReducer,
   },
 });
+
+export type RootState = ReturnType<typeof store.getState>;
+export default store;
+
+export type AppDispatch = typeof store.dispatch;
```

Comment:
This has been taken from the Redux Toolkit docs https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type